tests/conftest.py

```diff
@@ -55,13 +54,7 @@ class Course(Base):
         name = sa.Column(sa.String(255), nullable=False)
         # These are for better model form testing
         cost = sa.Column(sa.Numeric(5, 2), nullable=False)
-        description = sa.Column(
-            sa.Text,
-            nullable=True,
-            info=dict(
-                marshmallow=dict(validate=[validate.Length(max=1000)], required=True)
-            ),
-        )
+        description = sa.Column(sa.Text, nullable=True)
```

Comment:
This usage is depreciated, so I've removed from here and I add a specific column to the model for the test of this specific behavior..